sql/log_event.cc

```diff
@@ -5000,7 +5000,7 @@ int Query_log_event::do_apply_event(Relay_log_info const *rli,
         thd->enable_slow_log is set to the value of
         opt_log_slow_admin_statements).
       */
-      thd->enable_slow_log= TRUE;
+      thd->enable_slow_log= opt_log_slow_slave_statements;
```

Comment:
what about the other place mentioned by bug reporter?
sql/slave.cc init_slave_thread

Reply:
It's already up to date with upstream
`  thd->slave_thread = 1;
  thd->enable_slow_log= opt_log_slow_slave_statements;
`

Reply:
ok :+1: